src/orders/template.yaml

```diff
@@ -5,7 +5,7 @@ Description: >
 
 Globals:
   Function:
-    Timeout: 3
+    Timeout: 7
```

Comment:
is this seconds? if its is we might want to increase it to 120 just to make sure we dont get a timeout